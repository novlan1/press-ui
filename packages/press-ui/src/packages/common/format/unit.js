import { isDef, IN_BROWSER, isNumeric } from '../utils/validator';

export function addUnit(value) {
  if (!isDef(value)) {
    return undefined;
  }

  value = String(value);
  return isNumeric(value) ? `${value}px` : value;
} // cache

let rootFontSize;

function getRootFontSize() {
  if (!rootFontSize) {
    const doc = document.documentElement;
    const fontSize = doc.style.fontSize || window.getComputedStyle(doc).fontSize;
    rootFontSize = parseFloat(fontSize);
  }

  return rootFontSize;
}

function convertRem(value) {
  value = value.replace(/rem/g, '');
  return +value * getRootFontSize();
}

function convertVw(value) {
  value = value.replace(/vw/g, '');
  return +value * window.innerWidth / 100;
}

function convertVh(value) {
  value = value.replace(/vh/g, '');
  return +value * window.innerHeight / 100;
}

export function unitToPx(value) {
  if (typeof value === 'number') {
    return value;
  }

  if (IN_BROWSER) {
    if (value.indexOf('rem') !== -1) {
      return convertRem(value);
    }

    if (value.indexOf('vw') !== -1) {
      return convertVw(value);
    }

    if (value.indexOf('vh') !== -1) {
      return convertVh(value);
    }
  }

  return parseFloat(value);
}
