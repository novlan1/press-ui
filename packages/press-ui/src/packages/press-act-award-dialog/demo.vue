<template>
  <div
    class="demo-wrap"
    :class="curGameClass"
  >
    <ActDemo
      :custom-style-str="styleMap[customStyle].label"
      :show.sync="show"
      :hide-tip-style.sync="hideTipStyle"
      :use-tip-class.sync="useTipClass"
      @onCustom="onCustom"
    >
      <PressActAwardDialog
        sub-btn-text="订阅"
        :show.sync="show"
        :hide-tip-style.sync="hideTipStyle"
        :use-tip-class.sync="useTipClass"
        :issubscribe="isSubscribe"
        @clickSub="isSubscribe = !isSubscribe"
        @clickButton="show = false"
      />
    </ActDemo>

    <PressPopupCell
      :id="FUNCTIONAL_ID_MAP.POPUP_CELL"
      :ref="FUNCTIONAL_ID_MAP.POPUP_CELL"
      mode="functional"
    />

    <PressPicker
      :id="FUNCTIONAL_ID_MAP.PICKER"
      :ref="FUNCTIONAL_ID_MAP.PICKER"
      mode="functional"
    />
  </div>
</template>
<script>
import ActDemo from 'src/pages/components/act-demo/act-demo.vue';
import PressActAwardDialog from 'press-ui/press-act-award-dialog/press-act-award-dialog.vue';
// import PressActAwardDialog from 'src/lib/act-award-dialog';
import { getActCustomStyleMixin } from 'src/utils/mixin/act-custom-style.ts';
import { STYLE_LIST } from 'src/packages/press-act-award-dialog/demo-helper/style-list';
// import 'src/lib/lib-style/act-award-dialog.css';

export default {
  components: {
    PressActAwardDialog,
    ActDemo,
  },
  mixins: [
    getActCustomStyleMixin('PressActAwardDialog', STYLE_LIST),
  ],
  data() {
    return {
      show: true,
      isSubscribe: true,
    };
  },

  mounted() {
  },
  methods: {
  },
};
</script>
<style scoped lang="scss">
</style>
