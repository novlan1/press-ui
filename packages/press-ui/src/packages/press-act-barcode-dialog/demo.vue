<template>
  <div
    class="demo-wrap"
    :class="curGameClass"
  >
    <ActDemo
      :custom-style-str="styleMap[customStyle].label"
      :show.sync="show"
      :hide-tip-style.sync="hideTipStyle"
      :use-tip-class.sync="useTipClass"
      @onCustom="onCustom"
    >
      <PressActBarcodeDialog
        :show.sync="show"
        :hide-tip-style.sync="hideTipStyle"
        :use-tip-class.sync="useTipClass"
        :url="url"
      />
    </ActDemo>

    <PressPopupCell
      :id="FUNCTIONAL_ID_MAP.POPUP_CELL"
      :ref="FUNCTIONAL_ID_MAP.POPUP_CELL"
      mode="functional"
    />

    <PressPicker
      :id="FUNCTIONAL_ID_MAP.PICKER"
      :ref="FUNCTIONAL_ID_MAP.PICKER"
      mode="functional"
    />
  </div>
</template>
<script>
import ActDemo from 'src/pages/components/act-demo/act-demo.vue';
import PressActBarcodeDialog from 'press-ui/press-act-barcode-dialog/press-act-barcode-dialog.vue';
import { getActCustomStyleMixin } from 'src/utils/mixin/act-custom-style.ts';
import { STYLE_LIST } from 'src/packages/press-act-barcode-dialog/demo-helper/style-list';


export default {
  components: {
    PressActBarcodeDialog,
    ActDemo,
  },
  mixins: [
    getActCustomStyleMixin('PressActBarcodeDialog', STYLE_LIST),
  ],
  data() {
    return {
      show: true,
      url: 'https://www.qq.com',
    };
  },

  mounted() {
  },
  methods: {
  },
};
</script>
<style scoped lang="scss">
</style>
