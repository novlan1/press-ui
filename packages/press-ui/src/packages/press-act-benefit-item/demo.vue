<template>
  <div
    class="demo-wrap"
    :class="curGameClass"
  >
    <div class="component-wrap">
      <PressActBenefitItem
        :hide-tip-style.sync="hideTipStyle"
        :use-tip-class.sync="useTipClass"
        @clickGiftDetail="clickGiftDetail"
        @clickGetTaskReward="clickGetTaskReward"
        @clickMerc="clickMerc"
      />
    </div>

    <ActDemo
      :custom-style-str="styleMap[customStyle].label"
      :hide-tip-style.sync="hideTipStyle"
      :use-tip-class.sync="useTipClass"
      :show-basic-usage="false"
      @onCustom="onCustom"
    />

    <PressPopupCell
      :id="FUNCTIONAL_ID_MAP.POPUP_CELL"
      :ref="FUNCTIONAL_ID_MAP.POPUP_CELL"
      mode="functional"
    />

    <PressPicker
      :id="FUNCTIONAL_ID_MAP.PICKER"
      :ref="FUNCTIONAL_ID_MAP.PICKER"
      mode="functional"
    />
  </div>
</template>
<script>
import PressActBenefitItem from 'press-ui/press-act-benefit-item/press-act-benefit-item.vue';
import { getActCustomStyleMixin } from 'src/utils/mixin/act-custom-style.ts';
import { STYLE_LIST } from 'src/packages/press-act-benefit-item/demo-helper/style-list';
import ActDemo from 'src/pages/components/act-demo/act-demo.vue';


export default {
  components: {
    PressActBenefitItem,
    ActDemo,
  },
  mixins: [
    getActCustomStyleMixin('PressActBenefitItem', STYLE_LIST, 'pvp-vertical'),
  ],
  data() {
    return {
    };
  },

  mounted() {
  },
  methods: {
    clickGiftDetail() {
      this.onGTip('[clickGiftDetail]');
    },
    clickGetTaskReward() {
      this.onGTip('[clickGetTaskReward]');
    },
    clickMerc() {
      this.onGTip('[clickMerc]');
    },
  },
};
</script>
<style scoped lang="scss">
.component-wrap {
  margin: 20px 0 50px;
}
</style>
