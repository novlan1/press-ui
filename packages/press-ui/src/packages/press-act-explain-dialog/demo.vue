<template>
  <div
    class="demo-wrap"
    :class="curGameClass"
  >
    <ActDemo
      :custom-style-str="styleMap[customStyle].label"
      :show.sync="show"
      :hide-tip-style.sync="hideTipStyle"
      :use-tip-class.sync="useTipClass"
      @onCustom="onCustom"
    >
      <PressActExplainDialog
        :show.sync="show"
        :show-button="true"
        :hide-tip-style.sync="hideTipStyle"
        :use-tip-class.sync="useTipClass"
        :props-data="propsData"
        :hide-base-style="customStyle !== CUSTOM_STYLE_DEFAULT"
      />
    </ActDemo>

    <PressPopupCell
      :id="FUNCTIONAL_ID_MAP.POPUP_CELL"
      :ref="FUNCTIONAL_ID_MAP.POPUP_CELL"
      mode="functional"
    />

    <PressPicker
      :id="FUNCTIONAL_ID_MAP.PICKER"
      :ref="FUNCTIONAL_ID_MAP.PICKER"
      mode="functional"
    />
  </div>
</template>
<script>
import ActDemo from 'src/pages/components/act-demo/act-demo.vue';
import PressActExplainDialog from 'press-ui/press-act-explain-dialog/press-act-explain-dialog.vue';
import { getActCustomStyleMixin } from 'src/utils/mixin/act-custom-style.ts';
import { STYLE_LIST } from 'src/packages/press-act-explain-dialog/demo-helper/style-list';


const PROPS_DATA = {
  title: '标题',
  content: Array.from({ length: 10 }).map((_, index) => `${index + 1}. Press UI 是一套支持 uni-app、普通 Vue 项目，同时兼容 Vue2 和 Vue3 的组件库`)
    .join('<br/>'),
};


export default {
  components: {
    PressActExplainDialog,
    ActDemo,
  },
  mixins: [
    getActCustomStyleMixin('PressActExplainDialog', STYLE_LIST, 'pvp-vertical'),
  ],
  data() {
    return {
      show: true,
      propsData: PROPS_DATA,
    };
  },

  mounted() {
  },
  methods: {
  },
};
</script>
<style scoped lang="scss">
</style>
