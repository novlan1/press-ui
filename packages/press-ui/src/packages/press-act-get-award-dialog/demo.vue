<template>
  <div
    class="demo-wrap"
    :class="curGameClass"
  >
    <ActDemo
      :custom-style-str="styleMap[customStyle].label"
      :show.sync="show"
      :hide-tip-style.sync="hideTipStyle"
      :use-tip-class.sync="useTipClass"
      @onCustom="onCustom"
    >
      <PressActGetAwardDialog
        :show.sync="show"
        :hide-tip-style="hideTipStyle"
        :use-tip-class.sync="useTipClass"
        :hide-base-style="customStyle !== CUSTOM_STYLE_DEFAULT"
      />
    </ActDemo>

    <PressPopupCell
      :id="FUNCTIONAL_ID_MAP.POPUP_CELL"
      :ref="FUNCTIONAL_ID_MAP.POPUP_CELL"
      mode="functional"
    />

    <PressPicker
      :id="FUNCTIONAL_ID_MAP.PICKER"
      :ref="FUNCTIONAL_ID_MAP.PICKER"
      mode="functional"
    />
  </div>
</template>
<script>
import ActDemo from 'src/pages/components/act-demo/act-demo.vue';
import PressActGetAwardDialog from 'press-ui/press-act-get-award-dialog/press-act-get-award-dialog.vue';
import { getActCustomStyleMixin } from 'src/utils/mixin/act-custom-style.ts';
import { STYLE_LIST } from 'src/packages/press-act-get-award-dialog/demo-helper/style-list';


export default {
  components: {
    PressActGetAwardDialog,
    ActDemo,
  },
  mixins: [
    getActCustomStyleMixin('PressActGetAwardDialog', STYLE_LIST),
  ],
  data() {
    return {
      show: true,
    };
  },

  mounted() {
  },
  methods: {
  },
};
</script>
<style scoped lang="scss">
</style>
