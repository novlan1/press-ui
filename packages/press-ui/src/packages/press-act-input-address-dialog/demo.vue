<template>
  <div
    class="demo-wrap"
    :class="curGameClass"
  >
    <ActDemo
      :custom-style-str="styleMap[customStyle].label"
      :show.sync="show"
      :hide-tip-style.sync="hideTipStyle"
      :use-tip-class.sync="useTipClass"
      @onCustom="onCustom"
    >
      <PressActInputAddressDialog
        :show.sync="show"
        :hide-tip-style.sync="hideTipStyle"
        :use-tip-class.sync="useTipClass"
        :props-data.sync="propsData"
        @clickButton="clickButton"
      />
    </ActDemo>

    <PressPopupCell
      :id="FUNCTIONAL_ID_MAP.POPUP_CELL"
      :ref="FUNCTIONAL_ID_MAP.POPUP_CELL"
      mode="functional"
    />

    <PressPicker
      :id="FUNCTIONAL_ID_MAP.PICKER"
      :ref="FUNCTIONAL_ID_MAP.PICKER"
      mode="functional"
    />
  </div>
</template>
<script>
import ActDemo from 'src/pages/components/act-demo/act-demo.vue';
import PressActInputAddressDialog from 'press-ui/press-act-input-address-dialog/press-act-input-address-dialog.vue';
import { getActCustomStyleMixin } from 'src/utils/mixin/act-custom-style.ts';
import { STYLE_LIST } from 'src/packages/press-act-input-address-dialog/demo-helper/style-list';

const PROPS_DATA = {
  name: '名字',
  mobile: '123456789',
  zipcode: '888888',
  district: '地区1',
  detail_addr: '详细地址:',
  province: '',
  city: '',
};

export default {
  components: {
    PressActInputAddressDialog,
    ActDemo,
  },
  mixins: [
    getActCustomStyleMixin('PressActInputAddressDialog', STYLE_LIST, 'pvp-vertical'),
  ],
  data() {
    return {
      show: true,
      propsData: PROPS_DATA,
    };
  },

  mounted() {
  },
  methods: {
    clickButton() {
      console.log('[clickButton]', this.propsData);
      this.show = false;
    },
  },
};
</script>
<style scoped lang="scss">
</style>
