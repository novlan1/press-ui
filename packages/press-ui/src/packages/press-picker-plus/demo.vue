<template>
  <div
    class="demo-wrap"
  >
    <demo-block :title="t('basicUsage')">
      <press-picker-plus
        :columns="columns"
        @change="onChange"
      />
    </demo-block>

    <demo-block :title="t('defaultIndex')">
      <press-picker-plus
        :columns="columns"
        :default-index="2"
        @change="onChange"
      />
    </demo-block>

    <demo-block :title="t('showToolbar')">
      <press-picker-plus
        show-toolbar
        :title="t('title')"
        :columns="columns"
        @change="onChange"
        @cancel="onCancel"
        @confirm="onConfirm"
      />
    </demo-block>

    <demo-block :title="t('multipleColumns')">
      <press-picker-plus
        ref="picker"
        :columns="cColumns"
        @change="onChangePicker"
      />
    </demo-block>

    <demo-block :title="t('disableOption')">
      <press-picker-plus
        :columns="disabledColumns"
        @change="onChange"
      />
    </demo-block>

    <demo-block :title="t('loadingStatus')">
      <press-picker-plus
        :columns="disabledColumns"
        loading
        @change="onChange"
      />
    </demo-block>

    <demo-block
      :title="t('withPopupPlus')"
    >
      <press-cell
        :title="t('check')"
        is-link
        @click="onShowPopupPlus"
      />
    </demo-block>


    <PressPickerPlusPopupPlus
      :show="showPopupPlus"
      :columns="columns"
      @change="onChange"
      @cancel="showPopupPlus = false"
      @confirm="confirmPopupPlus"
    />
  </div>
</template>
<script>
import PressPickerPlus from 'press-ui/press-picker-plus/press-picker-plus.vue';
import PressPickerPlusPopupPlus from 'press-ui/press-picker-plus-popup-plus/press-picker-plus-popup-plus.vue';


const dateColumns = {
  'zh-CN': [
    {
      values: ['周一', '周二', '周三', '周四', '周五'],
      defaultIndex: 2,
    },
    {
      values: ['上午', '下午', '晚上'],
      defaultIndex: 1,
    },
  ],
  'en-US': [
    {
      values: ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday'],
      defaultIndex: 2,
    },
    {
      values: ['Morning', 'Afternoon', 'Evening'],
      defaultIndex: 1,
    },
  ],
};

const cascadeColumns = {
  'zh-CN': [
    {
      text: '浙江',
      children: [
        {
          text: '杭州',
          children: [{ text: '西湖区' }, { text: '余杭区' }],
        },
        {
          text: '温州',
          children: [{ text: '鹿城区' }, { text: '瓯海区' }],
        },
      ],
    },
    {
      text: '福建',
      children: [
        {
          text: '福州',
          children: [{ text: '鼓楼区' }, { text: '台江区' }],
        },
        {
          text: '厦门',
          children: [{ text: '思明区' }, { text: '海沧区' }],
        },
      ],
    },
  ],
  'en-US': [
    {
      text: 'Zhejiang',
      children: [
        {
          text: 'Hangzhou',
          children: [{ text: 'Xihu' }, { text: 'Yuhang' }],
        },
        {
          text: 'Wenzhou',
          children: [{ text: 'Lucheng' }, { text: 'Ouhai' }],
        },
      ],
    },
    {
      text: 'Fujian',
      children: [
        {
          text: 'Fuzhou',
          children: [{ text: 'Gulou' }, { text: 'Taijiang' }],
        },
        {
          text: 'Xiamen',
          children: [{ text: 'Siming' }, { text: 'Haicang' }],
        },
      ],
    },
  ],
};

const tagHtml = `
<div style="position: relative;width: 100%;height: 100%;display:flex; align-items:center;justify-content: center;">杭州
 <span style="position: absolute; top: 0; left: calc(50% + 19px);color: #fff;font-size: 10px;background: rgba(83, 0, 195, 0.75);width: 26px; height: 13px;line-height: 13px;">常用</span>
</div>
`;

export default {
  i18n: {
    'zh-CN': {
      city: '城市',
      cascade: '级联选择',
      withPopup: '搭配弹出层使用',
      chooseCity: '选择城市',
      showToolbar: '展示顶部栏',
      dateColumns: dateColumns['zh-CN'],
      defaultIndex: '默认选中项',
      disableOption: '禁用选项',
      cascadeColumns: cascadeColumns['zh-CN'],
      multipleColumns: '多列联动',
      setColumnValues: '动态设置选项',
      textColumns: [
        '杭州',
        '宁波',
        '温州',
        '绍兴',
        '湖州',
        '嘉兴',
        '金华',
        '衢州',
      ],
      disabledColumns: [
        { text: '杭州', disabled: true },
        { text: '宁波' },
        { text: '温州' },
      ],
      column3: {
        浙江: [
          {
            html: tagHtml,
            text: '杭州',
          },
          '宁波', '温州', '嘉兴', '湖州',
        ],
        福建: ['福州', '厦门', '莆田', '三明', '泉州'],
      },
      toastContent: (value, index) => `当前值：${value}, 当前索引：${index}`,
      withPopupPlus: '结合 PopupPlus',
    },
    'en-US': {
      city: 'City',
      cascade: 'Cascade',
      withPopup: 'With Popup',
      chooseCity: 'Choose City',
      showToolbar: 'Show Toolbar',
      dateColumns: dateColumns['en-US'],
      defaultIndex: 'Default Index',
      disableOption: 'Disable Option',
      cascadeColumns: cascadeColumns['en-US'],
      multipleColumns: 'Multiple Columns',
      setColumnValues: 'Set Column Values',
      textColumns: ['Delaware', 'Florida', 'Georqia', 'Indiana', 'Maine'],
      disabledColumns: [
        { text: 'Delaware', disabled: true },
        { text: 'Florida' },
        { text: 'Georqia' },
      ],
      column3: {
        Group1: ['Delaware', 'Florida', 'Georqia', 'Indiana', 'Maine'],
        Group2: ['Alabama', 'Kansas', 'Louisiana', 'Texas'],
      },
      toastContent: (value, index) => `Value: ${value}, Index：${index}`,
      withPopupPlus: 'With PopupPlus',
    },
  },
  components: {
    PressPickerPlus,
    PressPickerPlusPopupPlus,
  },
  data() {
    return {
      showPopupPlus: false,
    };
  },
  computed: {
    columns() {
      return this.t('textColumns');
    },
    disabledColumns() {
      return this.t('disabledColumns');
    },
    cColumns() {
      const citys = this.t('column3');
      return [
        {
          values: Object.keys(citys),
          className: 'column1',
        },
        {
          values: Object.values(citys)[0],
          className: 'column2',
          defaultIndex: 2,
        },
      ];
    },
  },
  methods: {
    onChangePicker(val) {
      const cities = this.t('column3');
      const { value } = val;
      this.$refs.picker.setColumnValues(1, cities[value[0]]);
      const values = this.$refs.picker.getValues();
      const indexes = this.$refs.picker.getIndexes();
      this.onTip({
        value: values.map(item => item.text || item),
        index: indexes,
      });
    },
    onChange(val) {
      console.log('onChange.val', val);
      this.onTip(val);
    },
    onCancel(val) {
      console.log('onCancel.val', val);
      this.onTip(val, '[cancel] ');
    },
    onConfirm(val) {
      console.log('onConfirm.val', val);
      this.onTip(val, '[confirm] ');
    },
    onTip(val, prefix = '') {
      const { value, index } = val;
      const title = `${prefix}value: ${value.text || value}, index: ${index}`;
      this.onGTip(title);
    },
    onShowPopupPlus() {
      this.showPopupPlus = true;
    },
    confirmPopupPlus(...args) {
      console.log('[confirmPopupPlus]', ...args);
      this.showPopupPlus = false;
    },
  },
};
</script>

<style scoped lang="scss">
</style>

