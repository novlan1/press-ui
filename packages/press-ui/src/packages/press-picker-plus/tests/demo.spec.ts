import Demo from '../demo.vue';
import { snapshotDemo } from '../../../../tests/unit/demo';


snapshotDemo(Demo);
