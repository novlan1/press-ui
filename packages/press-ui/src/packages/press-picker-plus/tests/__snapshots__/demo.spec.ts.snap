// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders demo correctly 1`] = `
<div class="demo-wrap">
  <div>
    <div class="press-picker-index">
      <div class="press-picker">
        <div class="press-picker-toolbar">
          <!---->
        </div>
        <!---->
        <div class="press-picker__columns" style="height: 264px;">
          <div class="press-picker-column-index press-picker__column" data-index="0">
            <div class="press-picker-column" style="height: 264px;">
              <div class="press-picker-column" style="height: 264px;">
                <div style="transition: transform 0ms; line-height: 44px; transform: translate3d(0, 110px, 0);">
                  <div data-index="0" class="press-ellipsis press-picker-column__item press-picker-column__item--selected activeClass" style="height: 44px; line-height: 44px;">
                    杭州
                  </div>
                  <div data-index="1" class="press-ellipsis press-picker-column__item" style="height: 44px; line-height: 44px;">
                    宁波
                  </div>
                  <div data-index="2" class="press-ellipsis press-picker-column__item" style="height: 44px; line-height: 44px;">
                    温州
                  </div>
                  <div data-index="3" class="press-ellipsis press-picker-column__item" style="height: 44px; line-height: 44px;">
                    绍兴
                  </div>
                  <div data-index="4" class="press-ellipsis press-picker-column__item" style="height: 44px; line-height: 44px;">
                    湖州
                  </div>
                  <div data-index="5" class="press-ellipsis press-picker-column__item" style="height: 44px; line-height: 44px;">
                    嘉兴
                  </div>
                  <div data-index="6" class="press-ellipsis press-picker-column__item" style="height: 44px; line-height: 44px;">
                    金华
                  </div>
                  <div data-index="7" class="press-ellipsis press-picker-column__item" style="height: 44px; line-height: 44px;">
                    衢州
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div class="press-picker__mask" style="background-size: 100% 110px;"></div>
          <div class="press-picker__frame press-hairline--top-bottom" style="height: 44px;"></div>
        </div>
        <!---->
      </div>
    </div>
  </div>
  <div>
    <div class="press-picker-index">
      <div class="press-picker">
        <div class="press-picker-toolbar">
          <!---->
        </div>
        <!---->
        <div class="press-picker__columns" style="height: 264px;">
          <div class="press-picker-column-index press-picker__column" data-index="0">
            <div class="press-picker-column" style="height: 264px;">
              <div class="press-picker-column" style="height: 264px;">
                <div style="transition: transform 0ms; line-height: 44px; transform: translate3d(0, 110px, 0);">
                  <div data-index="0" class="press-ellipsis press-picker-column__item" style="height: 44px; line-height: 44px;">
                    杭州
                  </div>
                  <div data-index="1" class="press-ellipsis press-picker-column__item" style="height: 44px; line-height: 44px;">
                    宁波
                  </div>
                  <div data-index="2" class="press-ellipsis press-picker-column__item press-picker-column__item--selected activeClass" style="height: 44px; line-height: 44px;">
                    温州
                  </div>
                  <div data-index="3" class="press-ellipsis press-picker-column__item" style="height: 44px; line-height: 44px;">
                    绍兴
                  </div>
                  <div data-index="4" class="press-ellipsis press-picker-column__item" style="height: 44px; line-height: 44px;">
                    湖州
                  </div>
                  <div data-index="5" class="press-ellipsis press-picker-column__item" style="height: 44px; line-height: 44px;">
                    嘉兴
                  </div>
                  <div data-index="6" class="press-ellipsis press-picker-column__item" style="height: 44px; line-height: 44px;">
                    金华
                  </div>
                  <div data-index="7" class="press-ellipsis press-picker-column__item" style="height: 44px; line-height: 44px;">
                    衢州
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div class="press-picker__mask" style="background-size: 100% 110px;"></div>
          <div class="press-picker__frame press-hairline--top-bottom" style="height: 44px;"></div>
        </div>
        <!---->
      </div>
    </div>
  </div>
  <div>
    <div class="press-picker-index">
      <div class="press-picker">
        <div class="press-picker-toolbar">
          <div class="press-picker__toolbar toolbar-class">
            <div hover-class="press-picker__cancel--hover" hover-stay-time="70" data-type="cancel" class="press-picker__cancel">
              取消
            </div>
            <!---->
            <div hover-class="press-picker__confirm--hover" hover-stay-time="70" data-type="confirm" class="press-picker__confirm">
              确定
            </div>
          </div>
        </div>
        <!---->
        <div class="press-picker__columns" style="height: 264px;">
          <div class="press-picker-column-index press-picker__column" data-index="0">
            <div class="press-picker-column" style="height: 264px;">
              <div class="press-picker-column" style="height: 264px;">
                <div style="transition: transform 0ms; line-height: 44px; transform: translate3d(0, 110px, 0);">
                  <div data-index="0" class="press-ellipsis press-picker-column__item press-picker-column__item--selected activeClass" style="height: 44px; line-height: 44px;">
                    杭州
                  </div>
                  <div data-index="1" class="press-ellipsis press-picker-column__item" style="height: 44px; line-height: 44px;">
                    宁波
                  </div>
                  <div data-index="2" class="press-ellipsis press-picker-column__item" style="height: 44px; line-height: 44px;">
                    温州
                  </div>
                  <div data-index="3" class="press-ellipsis press-picker-column__item" style="height: 44px; line-height: 44px;">
                    绍兴
                  </div>
                  <div data-index="4" class="press-ellipsis press-picker-column__item" style="height: 44px; line-height: 44px;">
                    湖州
                  </div>
                  <div data-index="5" class="press-ellipsis press-picker-column__item" style="height: 44px; line-height: 44px;">
                    嘉兴
                  </div>
                  <div data-index="6" class="press-ellipsis press-picker-column__item" style="height: 44px; line-height: 44px;">
                    金华
                  </div>
                  <div data-index="7" class="press-ellipsis press-picker-column__item" style="height: 44px; line-height: 44px;">
                    衢州
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div class="press-picker__mask" style="background-size: 100% 110px;"></div>
          <div class="press-picker__frame press-hairline--top-bottom" style="height: 44px;"></div>
        </div>
        <!---->
      </div>
    </div>
  </div>
  <div>
    <div class="press-picker-index">
      <div class="press-picker">
        <div class="press-picker-toolbar">
          <!---->
        </div>
        <!---->
        <div class="press-picker__columns" style="height: 264px;">
          <div class="press-picker-column-index press-picker__column" data-index="0">
            <div class="press-picker-column" style="height: 264px;">
              <div class="press-picker-column" style="height: 264px;">
                <div style="transition: transform 0ms; line-height: 44px; transform: translate3d(0, 110px, 0);">
                  <div data-index="0" class="press-ellipsis press-picker-column__item press-picker-column__item--selected activeClass" style="height: 44px; line-height: 44px;">
                    浙江
                  </div>
                  <div data-index="1" class="press-ellipsis press-picker-column__item" style="height: 44px; line-height: 44px;">
                    福建
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div class="press-picker-column-index press-picker__column" data-index="1">
            <div class="press-picker-column" style="height: 264px;">
              <div class="press-picker-column" style="height: 264px;">
                <div style="transition: transform 0ms; line-height: 44px; transform: translate3d(0, 110px, 0);">
                  <div data-index="0" class="press-ellipsis press-picker-column__item" style="height: 44px; line-height: 44px;">
                    杭州
                  </div>
                  <div data-index="1" class="press-ellipsis press-picker-column__item" style="height: 44px; line-height: 44px;">
                    宁波
                  </div>
                  <div data-index="2" class="press-ellipsis press-picker-column__item press-picker-column__item--selected activeClass" style="height: 44px; line-height: 44px;">
                    温州
                  </div>
                  <div data-index="3" class="press-ellipsis press-picker-column__item" style="height: 44px; line-height: 44px;">
                    嘉兴
                  </div>
                  <div data-index="4" class="press-ellipsis press-picker-column__item" style="height: 44px; line-height: 44px;">
                    湖州
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div class="press-picker__mask" style="background-size: 100% 110px;"></div>
          <div class="press-picker__frame press-hairline--top-bottom" style="height: 44px;"></div>
        </div>
        <!---->
      </div>
    </div>
  </div>
  <div>
    <div class="press-picker-index">
      <div class="press-picker">
        <div class="press-picker-toolbar">
          <!---->
        </div>
        <!---->
        <div class="press-picker__columns" style="height: 264px;">
          <div class="press-picker-column-index press-picker__column" data-index="0">
            <div class="press-picker-column" style="height: 264px;">
              <div class="press-picker-column" style="height: 264px;">
                <div style="transition: transform 0ms; line-height: 44px; transform: translate3d(0, 110px, 0);">
                  <div data-index="0" class="press-ellipsis press-picker-column__item press-picker-column__item--disabled press-picker-column__item--selected activeClass" style="height: 44px; line-height: 44px;">
                    杭州
                  </div>
                  <div data-index="1" class="press-ellipsis press-picker-column__item" style="height: 44px; line-height: 44px;">
                    宁波
                  </div>
                  <div data-index="2" class="press-ellipsis press-picker-column__item" style="height: 44px; line-height: 44px;">
                    温州
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div class="press-picker__mask" style="background-size: 100% 110px;"></div>
          <div class="press-picker__frame press-hairline--top-bottom" style="height: 44px;"></div>
        </div>
        <!---->
      </div>
    </div>
  </div>
  <div>
    <div class="press-picker-index">
      <div class="press-picker">
        <div class="press-picker-toolbar">
          <!---->
        </div>
        <div class="press-picker__loading">
          <div class="press-loading-index">
            <div class=" press-loading">
              <div class="press-loading__spinner press-loading__spinner--circular" style="color: rgb(25, 137, 250);">
                <!---->
              </div>
              <div class="press-loading__text"></div>
            </div>
          </div>
        </div>
        <div class="press-picker__columns" style="height: 264px;">
          <div class="press-picker-column-index press-picker__column" data-index="0">
            <div class="press-picker-column" style="height: 264px;">
              <div class="press-picker-column" style="height: 264px;">
                <div style="transition: transform 0ms; line-height: 44px; transform: translate3d(0, 110px, 0);">
                  <div data-index="0" class="press-ellipsis press-picker-column__item press-picker-column__item--disabled press-picker-column__item--selected activeClass" style="height: 44px; line-height: 44px;">
                    杭州
                  </div>
                  <div data-index="1" class="press-ellipsis press-picker-column__item" style="height: 44px; line-height: 44px;">
                    宁波
                  </div>
                  <div data-index="2" class="press-ellipsis press-picker-column__item" style="height: 44px; line-height: 44px;">
                    温州
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div class="press-picker__mask" style="background-size: 100% 110px;"></div>
          <div class="press-picker__frame press-hairline--top-bottom" style="height: 44px;"></div>
        </div>
        <!---->
      </div>
    </div>
  </div>
  <div>
    <press-cell title="" is-link=""></press-cell>
  </div>
  <div class="press-popup-plus ">
    <div class="press-overlay-index">
      <!---->
    </div>
    <!---->
  </div>
</div>
`;
